reposerver/src/main/scala/com/advancedtelematic/tuf/reposerver/http/RoleRefresh.scala

```diff
@@ -41,15 +43,18 @@ class RepoRoleRefresh(signFn: RoleSigner)(implicit val db: Database, val ec: Exe
     val existingTargets = await(findExisting[TargetsRole](repoId))
     val existingSnapshots = await(findExisting[SnapshotRole](repoId))
     val existingTimestamps = await(findExisting[TimestampRole](repoId))
-    val (newTargets, dependencies) = await(roleRefresh.refreshTargets(existingTargets, existingTimestamps, existingSnapshots))
+    val delegations = await(delegationsManagement.findDelegations(existingTargets))
```

Comment:
Maybe the issue is the name. It is not finding delegations, but metadata from delegated roles. I didn't put getting the delegated roles into RoleRefresh since that doesn't use the DB currently. I do agree that in general the code using `findExisting` is a little repetitive.